### DISCUSSION TITLE
Make navigating small and/or whitespace changes easier

### DISCUSSION THREAD

**Discussion by includesec-erik** [outline/outline#5381] [Ideas]
A common problem we have is stray small edits in Outline

Sometimes you're on an Outline page and you accidently hit a \r\n or a \s while on the page and not realize it.

All of those subscribed to the page for updates receive a "Page Blah Blab updated" with a blank looking update (see screen shot below)
![image](https://github.com/outline/outline/assets/105012397/b00d681f-1a08-40f5-81e4-609b3c353b02)

And then when one goes to visit the page and try to find where the stray space was made, the change history/diff view doesn't make it obvious where the change is.

There's a couple of ways to improve this (#1 is the most complete solution IMHO)
1) Provide a feature (automatic, or a button/link) that allows pointers to all the changed lines to be highlighted in a meaningful way (perhaps at the right or left side bar of the line that was changed?)
2) Provide an option not to receive updates that are whitespace only
3) ???

**Comment by tommoor** [comment]
We should filter out these notifications as they are useless, there is some attempt at this but clearly not enough

**Reply by includesec-erik** [reply]
I see your point @tommoor, but for keeping the integrity of the docs, we should also be aware of exactly where that change was made to be able to go back and clean up the stray edit.

For us this situation happens, once or twice a month. Not enough to be a major problem, but often enough that these stray edits would affect the quality of the docs over time if they aren't noticed and reverted by the doc maintainers.

It feels like a medium difficulty UX problem!

**Reply by tommoor** [reply]
Another option, If your documents are largely read-only consider disabling "seamless editing" so that edits must be more purposeful.

<img width="698" alt="image" src="https://github.com/outline/outline/assets/380914/84e9b74d-0e42-4d70-a171-7f08506ed7f8">
